cinema/tests/test_order_api.py

```diff
@@ -82,6 +82,7 @@ def test_movie_session_detail_tickets(self):
 
     def test_movie_session_list_tickets_available(self):
         response = self.client.get(f"/api/cinema/movie_sessions/")
+        print(response.data)
```

Comment:
Remove the `print(response.data)` statement. Debug print statements should be removed from the final version of test cases to keep the output clean and focused on the test results.